app/components/interactive_dialog_controller/index.js

```diff
@@ -4,21 +4,24 @@
 import {bindActionCreators} from 'redux';
 import {connect} from 'react-redux';
 
+import {submitInteractiveDialog} from 'mattermost-redux/actions/integrations';
+
 import {showModal} from 'app/actions/navigation';
 
 import InteractiveDialogController from './interactive_dialog_controller';
 
 function mapStateToProps(state) {
     return {
         triggerId: state.entities.integrations.dialogTriggerId,
-        dialog: state.entities.integrations.dialog || {},
+        dialogData: state.entities.integrations.dialog || {},
```

Comment:
This change is for consistency and less confusing since dialogData also has `dialog` object inside.